doc/deploy/deployment.md

```diff
@@ -108,24 +108,50 @@ All Helm parameters are described in the table at the bottom of this section.
+helm upgrade --install --wait mcad-controller . \ 
+--namespace kube-system \ 
+--set image.repository=<image repository and name> \
+--set image.tag=<image tag> \ 
+--set imagePullSecret.name=<Name of image pull kubernetes secret> \ 
+--set imagePullSecret.password=<REPLACE_WITH_REGISTRY_TOKEN_GENERATED_IN_PREREQs_STAGE1_REGISTRY.d)>  \
+--set localConfigName=<Local Kubernetes Config File for Current Cluster>  \
+--set volumes.hostPath=<Host_Path_location_of_local_Kubernetes_config_file> \
+--set configMap.name=mcad-controller-configmap \
+--set configMap.quotaEnabled='"false"' \
+--set coscheduler.rbac.apiGroup="scheduling.sigs.k8s.io" \
+--set coscheduler.rbac.resource="podgroups"
 ```
 
 ##### Example 1
-*Assuming the default for `image.repository` and `image.tag` fields*:
+*Deploying MCAD using built images*:
+
 ```
-helm install mcad-controller --namespace kube-system
+helm upgrade --install --wait mcad . --namespace kube-system --set image.pullPolicy=IfNotPresent --set image.repository=darroyo/mcad-controller --set image.tag=quota-management-v1.29.40
+
 ```
+
 ##### Example 2
-*Assuming the MCAD controller image is already pulled onto the local target machine with the following image `image.repository=mcad-controller`, `image.tag=latest`*
+*Assuming the default for `image.repository` and `image.tag` fields*:
 ```
-helm install mcad-controller --namespace kube-system --wait --set image.pullPolicy=Never --set image.repository=mcad-controller --set image.tag=latest
+helm upgrade --install --wait mcad . --namespace kube-system
 ```
 ##### Example 3
+*Assuming the MCAD controller image is already pulled onto the local target machine with the following image `image.repository=mcad-controller`, `image.tag=latest`*
+```
+helm upgrade --install --wait mcad . --namespace kube-system --set image.pullPolicy=Never --set image.repository=mcad-controller --set image.tag=latest
+```
+##### Example 4
 To adjust the cpu and memory demands of the deployment with command line overrides example:
 
 ```
-helm install mcad-controller --namespace kube-system --wait --set resources.requests.cpu=1000m --set resources.requests.memory=1024Mi --set resources.limits.cpu=1000m --set resources.limits.memory=1024Mi --set image.repository=myDockerReegistry/mcad-controller --set image.tag=latest --set image.pullPolicy=Always
+helm upgrade --install --wait mcad . --namespace kube-system --wait --set resources.requests.cpu=1000m --set resources.requests.memory=1024Mi --set resources.limits.cpu=1000m --set resources.limits.memory=1024Mi --set image.repository=myDockerReegistry/mcad-controller --set image.tag=latest --set image.pullPolicy=Always
+```
+
+##### Example 5
```

Comment:
Should probably go after `3.b` or rename `3.b` to `6`? :)

Reply:
Not sure about the changes but examples 1-5 are related to section 3-a and then section 3-b starts